RELEASE.md

```diff
@@ -23,6 +23,14 @@
         `tf.keras.metrics.F1Score`, and `tf.keras.metrics.R2Score`.
     *   Added activation function `tf.keras.activations.mish`.
 
+*   Embeddings:
+
+    *   `tf.nn.embedding_lookup_sparse` and `tf.nn.safe_embedding_lookup_sparse`
+        now support ids and weights described by RaggedTensors, which are more
+        memory efficient than SparseTensors. The new boolean argument
+        `allow_fast_lookup` enables a simplified and typically faster lookup
+        procedure.
```

Comment:
```suggestion
    *   `tf.nn.embedding_lookup_sparse` and `tf.nn.safe_embedding_lookup_sparse`
        now support ids and weights described by `tf.RaggedTensor`.
    *   Added a new boolean argument `allow_fast_lookup` to 
        `tf.nn.embedding_lookup_sparse` and `tf.nn.safe_embedding_lookup_sparse`.
```

Reply:
A brief explanation of the purpose of `allow_fast_lookup` might be beneficial.

Reply:
Feel free to add it back.